scout/server/blueprints/variants/templates/variants/indicators.html

```diff
@@ -63,10 +63,22 @@
 
 {% macro evaluations_badge(variant) %}
   {% if variant.evaluations %}
-    {% for evaluation in (variant.evaluations or []) %}
-      <span class="badge bg-secondary" style="margin-left:1px" data-bs-toggle="tooltip" data-bs-placement="right"
-        title="Previously classified as {{ evaluation.classification.label }}">
-        {{ evaluation.classification.short }}
+    {# Create a dictionary to count classifications #}
+    {% set classification_counts = {} %}
+    {% for evaluation in variant.evaluations %}
```

Comment:
```suggestion
    {% for evaluation in (variant.evaluations or [])%}
```
Looked comfortable - don't know if it is really tested needed though.

Reply:
Ah, but isn't the kicker when it exists and is `None`? Anyway, I think you are still right - there are basically just four paths to having evaluations set, and all seem well initialised nowadays.

Reply:
I've tried adding a bunch of classifications and then I removed them all. It worked fine